lib/utils/RedisCommander.ts

```diff
@@ -8580,6 +8580,46 @@ interface RedisCommander<Context extends ClientContext = { type: "default" }> {
     callback?: Callback<unknown[]>
   ): Result<unknown[], Context>;
 
+  xautoclaimBuffer(
```

Comment:
It is simple copy-paste for non-buffer version typings - I assume it is okay, but let me know if it isn't...